java/create-terrain-from-a-local-tile-package/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,8 @@
+<resources>
+    <string name="app_name">Create Terrain From A Local Tile Package</string>
+
+    <string name="local_tile_package_location">/ArcGIS/Samples/tpk/MontereyElevation.tpk</string>
```

Comment:
/Samples/TileCache/MontereyElevation.tpk

Reply:
Is the full path `/ArcGIS/Samples/tpk/MontereyElevation.tpk`?